features/machine/machine.feature

```diff
@@ -595,3 +595,26 @@ Feature: Machine features testing
     Given I switch to cluster admin pseudo user
     Then evaluation of `cluster_operator('cloud-controller-manager').versions` is stored in the :versions clipboard
     And the expression should be true> cb.versions[0].include? "4"
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-33047
+  @admin
+  @destructive
+  Scenario: Machine API Operator should go degraded if any pod controller is crash looping
+    Given I have an IPI deployment
+    When I switch to cluster admin pseudo user
+    Then I use the "openshift-machine-api" project
+
+    And I run the steps 40 times or exit when co machine-api is degraded:
+    """
+    Given a pod becomes ready with labels:
+      | api=clusterapi     |
+      | k8s-app=controller |
+    When admin ensure "#{pod.name}" pod is deleted from the "openshift-machine-api" project
```

Comment:
Can we scale it down to 0 instead of repeatedly deleting the pod here?

Reply:
Tried it , but it doesnt report  , I mean the co unable to report it as Degraded always and brings up the deployment back to '1' , with killing pod we were able to get the degraded status everytime , hence using that .

Reply:
checked in 4.8 as well , it only works when we kill the pods .